### ISSUE TITLE
Installing @ton/blueprint==0.24.0 seems to be broken on NPM because of @tact-lang/compiler with NodeJS v22.4.1

### ISSUE THREAD

**Issue by imartemy1524** [tact-lang/tact#868]
### Are you using the latest released (or pre-released, a.k.a. "next") version?

- [X] I'm using the latest Tact version

### Relevant Tact/build system log output

```shell
941 verbose stack Error: command failed
941 verbose stack     at promiseSpawn (/home/im/.local/share/nvm/v22.4.1/lib/node_modules/npm/node_modules/@npmcli/promise-spawn/lib/index.js:22:22)
941 verbose stack     at spawnWithShell (/home/im/.local/share/nvm/v22.4.1/lib/node_modules/npm/node_modules/@npmcli/promise-spawn/lib/index.js:124:10)
941 verbose stack     at promiseSpawn (/home/im/.local/share/nvm/v22.4.1/lib/node_modules/npm/node_modules/@npmcli/promise-spawn/lib/index.js:12:12)
941 verbose stack     at runScriptPkg (/home/im/.local/share/nvm/v22.4.1/lib/node_modules/npm/node_modules/@npmcli/run-script/lib/run-script-pkg.js:77:13)
941 verbose stack     at runScript (/home/im/.local/share/nvm/v22.4.1/lib/node_modules/npm/node_modules/@npmcli/run-script/lib/run-script.js:9:12)
941 verbose stack     at /home/im/.local/share/nvm/v22.4.1/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/rebuild.js:329:17
941 verbose stack     at run (/home/im/.local/share/nvm/v22.4.1/lib/node_modules/npm/node_modules/promise-call-limit/dist/commonjs/index.js:67:22)
941 verbose stack     at /home/im/.local/share/nvm/v22.4.1/lib/node_modules/npm/node_modules/promise-call-limit/dist/commonjs/index.js:84:9
941 verbose stack     at new Promise (<anonymous>)
941 verbose stack     at callLimit (/home/im/.local/share/nvm/v22.4.1/lib/node_modules/npm/node_modules/promise-call-limit/dist/commonjs/index.js:35:69)
942 verbose pkgid @tact-lang/compiler@1.5.1
943 error code 127
944 error path /tmp/bob/node_modules/@tact-lang/compiler
945 error command failed
946 error command sh -c husky
```

### What happened?

NodeJS v22.4.1 NPM package manager failed to install latest @ton/blueprint version because of some weird error.

BTW, v19 succeed with this task, but it thew me this warning:
```
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@tact-lang/compiler@1.5.1',
npm WARN EBADENGINE   required: { node: '>=22.0.0' },
npm WARN EBADENGINE   current: { node: 'v19.9.0', npm: '9.6.3' }
npm WARN EBADENGINE }
```

`yarn`, `pnpm` and `bun` succeed with installing this package, it seems that the problem is only in NPM.

### What did you expect?

Installing blueprint for developing `func` contracts doesn't failed because of `tact` compiler package :)

### Steps to reproduce

```bash
docker run -it --rm node:22.4.1-bullseye bash
cd /tmp
echo "{\"dependencies\":{\"@ton/blueprint\": \"^0.24.0\"}}" > ./package.json
npm i
```

### How do you run Tact?

Blueprint

### Anything else?

Example of failed github actions unit test:
[bip](https://github.com/imartemy1524/ton-subdomain-manager/actions/runs/11022648737/job/30612418871)

**Comment by novusnota** [comment]
This is a known issue with npm — it tries to use development hooks even when not in the repository that enables them. Using `--ignore-scripts` is the workaround:

```shell
# If all your dependencies are in package.json
npm i --ignore-scripts

# If you're adding new dependencies on the CLI
npm i --ignore-scripts @ton/blueprint
npm i --ignore-scripts @tact-lang/compiler
``` 

Recently, I submitted a workaround to `create-ton`, which does exactly that. It's recommended to use it for [creating new Tact projects](https://docs.tact-lang.org/#start-2) and FunC projects as well.

**Comment by anton-trunov** [comment]
@imartemy1524 Thanks for reporting the issue. This should be fixed with Tact v1.5.2 https://www.npmjs.com/package/@tact-lang/compiler/v/1.5.2. Feel free to re-open the issue if it still does not work for you
